images/fcos-bastion-image/root/usr/bin/prepare_host_for_boot

```diff
@@ -41,11 +41,6 @@ until ipmi_cmd power status | grep -i -q "Chassis Power is off"; do
   sleep 30
 done
 
-if [[ "${ipxe_via_vmedia}" == "true" ]] && [[ "${BOOT_MODE}" == "pxe" ]]; then
```

Comment:
Hey Steve, we still need this, don't we? We should rather delete those lines that set the boot parameter in mount.vmedia.ipxe